core/src/main/scala/cats/arrow/Compose.scala

```diff
@@ -7,8 +7,11 @@ import simulacrum.typeclass
  * Must obey the laws defined in cats.laws.ComposeLaws.
  */
 @typeclass trait Compose[F[_, _]] { self =>
+
+  @simulacrum.op(":<<", alias = true)
```

Comment:
I also dislike it, but I am getting this compile error when introducing `>>:`, guess it's right associative, even when you put the `:` on the right.

```
[error] /Users/mark/Projects/cats/tests/src/test/scala/cats/tests/SyntaxTests.scala:55: type mismatch;
[error]  found   : x$1.type (with underlying type F[B,C])
[error]  required: F[D,?]
[error]     val a = x >>: y >>: z
[error]
```